src/pe/options.rs

```diff
@@ -16,6 +26,7 @@ impl ParseOptions {
         ParseOptions {
             resolve_rva: true,
             parse_attribute_certificates: true,
+            parse_mode: ParseMode::Strict,
         }
```

Comment:
can you also add a method that returns what the TE parser uses; we can call it `fn te()` or something like that, instead of manually constructing it in TE portion.